src/client/components/CompetitionCard.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import './styling/competition-card.css'
+
+function CompetitionCard() {
```

Comment:
Why not arrow functions?

Reply:
why not indeed

Reply:
App wasn't an arrow function, so I just kept the same. I do prefer arrow functions though, so happy to change them all for future components.

Reply:
I approve looking forward to seeing more arrow functions they are cooler